tests/rptest/tests/pandaproxy_test.py

```diff
@@ -122,6 +122,40 @@ def set_offsets(self,
                              headers=headers)
 
 
+class PandaProxyBrokersTest(RedpandaTest):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, num_brokers=4, **kwargs)
+
+    def setUp(self):
+        # Start the nodes manually for this test.
+        pass
+
+    @cluster(num_nodes=4)
+    def test_get_brokers_with_stale_health_metadata(self):
+        """
+        Test that a new member will be returned by /brokers
+        requests processed by followers even when the health metadata
+        is out of sync.
+        """
+        # Start first three nodes
+        self.redpanda.start(self.redpanda.nodes[0:-1])
```

Comment:
i think it would be more robust to slice the list in a way that would fail if other assumptions weren't met. maybe add a simple `assert len(nodes == 4)` at the beginning of the test?

```
>>> [0][0:-1]
[]
>>> [][0:-1]
[]
```

Reply:
Done. Thanks for the suggestion.